mllib-dal/src/main/scala/com/intel/oap/mllib/OneDAL.scala

```diff
@@ -399,7 +399,7 @@ object OneDAL {
                                     labelCol: String,
                                     featuresCol: String,
                                     executorNum: Int,
-                                    device: Common.ComputeDevice): RDD[(Long, Long)] = {
+                                    device: Common.ComputeDevice): RDD[(Tuple3[Long, Long, Long], Tuple3[Long, Long, Long])] = {
```

Comment:
add some comments for what is (Tuple3[Long, Long, Long], Tuple3[Long, Long, Long])